backend/internal/database/db.go

```diff
@@ -2,53 +2,79 @@ package database
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/CEM-KEA/whoknows/backend/internal/config"
 	"github.com/CEM-KEA/whoknows/backend/internal/models"
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/driver/postgres"
 	"gorm.io/driver/sqlite"
 	"gorm.io/gorm"
 )
 
 // DB is the variable that holds the database connection
 var DB *gorm.DB
 
-// InitDatabase creates a new database file if it does not exist and establishes a connection to it
+// InitDatabase initializes the Postgres database connection for production
 func InitDatabase() error {
-	databasePath := config.AppConfig.Database.FilePath
-
-	fmt.Printf("Connection to Database at: %s\n", databasePath)
-
+	dsn := fmt.Sprintf(
+		"host=%s user=%s password=%s dbname=%s port=%d sslmode=%s",
+		config.AppConfig.Database.Host,
+		config.AppConfig.Database.User,
+		config.AppConfig.Database.Password,
+		config.AppConfig.Database.Name,
+		config.AppConfig.Database.Port,
+		config.AppConfig.Database.SSLMode,
+	)
+	fmt.Printf("Attempting to connect to Postgres database with DSN: %s\n", dsn)
```

Comment:
Vi bør ikke udskrive password til log her

Reply:
du har ret! - det var for debugging og skal naturligvis fjernes